src/dpp/guild.cpp

```diff
@@ -347,6 +347,11 @@ bool guild_member::has_rejoined() const {
 	return flags & dpp::gm_did_rejoin;
 }
 
+bool guild_member::is_guild_owner() const {
+	auto* _guild = find_guild(user_id);
+	return _guild != nullptr && _guild->owner_id == this->user_id;
```

Comment:
So if the guild is not found in the cache this returns false, this is a bit weird, do we have any other functions in D++ that do this or is this potentially going to surprise people? @braindigitalis

Reply:
it might surprise people who have cache off yes

Reply:
it should probably be a method of dpp::guild that takes a user. not really needed though you can just check user id == guild owner id where you have both.

most people do have guild cache on. best to clearly document this assumption in the docs of the function.